phoonnx/model_manager.py

```
@@ -200,11 +384,15 @@ def get_piper_voice_list(self):
 
         for v in piper_voices.values():
             try:
+                voice_key = v["key"] 
+                display_name = self._generate_display_name(voice_key, v, Engine.PIPER)
+
                 voice = TTSModelInfo(
                     voice_id="piper_" + v["key"],
                     lang=standardize_tag(v["key"].split("-")[0]),
                     model_url=base + [a for a in v["files"] if a.endswith(".onnx")][0],
```

Ruff (0.14.1)

Prefer `next(a for a in v["files"] if a.endswith(".onnx"))` over single element slice

Replace with `next(a for a in v["files"] if a.endswith(".onnx"))`

(RUF015)